darts/models/forecasting/arima.py

```diff
@@ -32,10 +32,17 @@ class ARIMA(TransferableFutureCovariatesLocalForecastingModel):
     def __init__(
         self,
         p: int = 12,
-        d: int = 1,
-        q: int = 0,
-        seasonal_order: Tuple[int, int, int, int] = (0, 0, 0, 0),
-        trend: Optional[str] = None,
+        d: Union[int, Iterable[int]] = 1,
```

Comment:
Why are you using Iterable instead of List? Can you give an example where you don't have a list of integer (or an integer)?

Reply:
Oops, it should be a sequence (it is a subtype of Iterable but without sets or dicts). A sequence is compatible with tuples, numpy arrays, torch tensors and lists. It is more general than lists.

Reply:
Ah ok. I was reading the docstring and I was sure we could have just a list or an int. Perhaps update the docstring also

Reply:
fixed. Btw, I just checked and torch.Tensor are *not* sequences since

```python
def head(x: Sequence[int]) -> int:
    return x[0]
```

but

```python
def head(x: torch.IntTensor) -> torch.IntTensor:
    return x[0]
```

(from https://discuss.pytorch.org/t/why-is-tensor-not-subclass-of-collections-abc-sequence/110126/2)